Source:   KDE SVN, combined changes from r1102482 and r1145054
Upstream: Yes
Reason:   Compile fixes for Qt 4.7. (Can't initialize QString with an int anymore)

--- konq-plugins/adblock/adblock.cpp	2010/07/01 15:43:50	1145053
+++ konq-plugins/adblock/adblock.cpp	2010/07/01 15:44:18	1145054
@@ -300,7 +300,7 @@
 // ----------------------------------------------------------------------------
 
 AdElement::AdElement() :
-  m_url(0), m_category(0), m_type(0), m_blocked(false) {}
+  m_url(QString()), m_category(QString()), m_type(QString()), m_blocked(false) {}
 
 AdElement::AdElement(const QString &url, const QString &category,
 		     const QString &type, bool blocked, const DOM::Node&node) :
